sourceCode.py

```diff
@@ -1 +1,523 @@
+    
+    [r'Is there (.*)',
+     [  "Do you think there is %1?",
+      "It's likely that there is %1.",
+      "Would you like there to be %1?"]],
+    
+    [r'My (.*)',
+     [  "I see, your %1.",
+      "Why do you say that your %1?",
+      "When your %1, how do you feel?"]],
+    
+    [r'You (.*)',
+     [  "We should be discussing you, not me.",
+      "Why do you care whether I %1?"]],
+    
+    [r'Why (.*)',
+     [  "Why don't you tell me the reason why %1?",
+      "Why do you think %1?" ]],
+    
+    [r'I want (.*)',
+     [  "What would it mean to you if you got %1?",
+      "Why do you want %1?",
+      "What would you do if you got %1?",
+      "If you got %1, then what would you do?"]],
+    
+    [r'(.*) mother(.*)',
+     [  "Tell me more about your mother.",
+      "What was your relationship with your mother like?",
+      "How do you feel about your mother?",
+      "How does this relate to your feelings today?",
+      "Good family relations are important."]],
+    
+    [r'(.*) father(.*)',
+     [  "Tell me more about your father.",
+      "How did your father make you feel?",
+      "How do you feel about your father?",
+      "Does your relationship with your father relate to your feelings today?",
+      "Do you have trouble showing affection with your family?"]],
+    
+    [r'(.*) child(.*)',
+     [  "Did you have close friends as a child?",
+      "What is your favorite childhood memory?",
+      "Do you remember any dreams or nightmares from childhood?",
+      "Did the other children sometimes tease you?",
+      "How do you think your childhood experiences relate to your feelings today?"]],
+    
+    [r'(.*)\?',
+     [  "Why do you ask that?",
+      "Please consider whether you can answer your own question.",
+      "Perhaps the answer lies within yourself?",
+      "Why don't you tell me?"]],
+    
+    [r'quit',
+     [  "Thank you for talking with me.",
+      "Good-bye.",
+      "Thank you, that will be $150.  Have a good day!"]],
+    
+    [r'(.*)',
+     [  "Please tell me more.",
+      "Let's change focus a bit... Tell me about your family.",
+      "Very interesting.",
+      "%1.",
+      "I see.  And what does that tell you?",
+      "How does that make you feel?",
+      "How do you feel when you say that?"]]
+    ]
+
+## create function to talk with Jordan about your sorrows
+import random
+import re
+
+def caring_Jordan(userInput):
+    keys_caring = list(map(lambda x:re.compile(x[0], re.IGNORECASE), library_caring))
+    values_caring = list(map(lambda x:x[1],library_caring))
+    
+    while True:
+        userInput = input(">>> ")
+        for i in range(0, len(keys_caring)):
+            match = keys_caring[i].match(userInput)
+            if match:
+                resp = random.choice(values_caring[i])
+                pos = resp.find('%')
+                print(resp)
+                else:
+                    print("I did not understand what you said")
```

Comment:
You forgot to indent these lines
```suggestion
            else:
                print("I did not understand what you said")
```
But if you agree with the suggestion I made above then I think you can use it here as well! Otherwise it prints "I did not understand what you said" as many times as len(keys_caring)